src/dmd/dsymbolsem.d

```diff
@@ -2991,13 +2991,13 @@ private extern(C++) final class DsymbolSemanticVisitor : Visitor
 
                     // This function overrides fdv
                     if (fdv.isFinalFunc())
-                        funcdecl.error("cannot override final function `%s`", fdv.toPrettyChars());
+                        funcdecl.error("cannot override `final` function `%s`", fdv.toPrettyChars());
 
                     if (!funcdecl.isOverride())
                     {
                         if (fdv.isFuture())
                         {
-                            deprecation(funcdecl.loc, "@future base class method %s is being overridden by %s; rename the latter", fdv.toPrettyChars(), funcdecl.toPrettyChars());
+                            deprecation(funcdecl.loc, "`@future` base class method `%s` is being overridden by `%s`; rename the latter", fdv.toPrettyChars(), funcdecl.toPrettyChars());
```

Comment:
@future or @__future? (I wasn't in discussions when this feature was implemented).

Reply:
You're right, it's `@__future`. Good catch!

Reply:
Fixed